Userland/Services/WindowServer/WMClientConnection.h

```diff
@@ -47,6 +47,7 @@ class WMClientConnection final
     virtual void handle(const Messages::WindowManagerServer::StartWindowResize&) override;
     virtual void handle(const Messages::WindowManagerServer::PopupWindowMenu&) override;
     virtual void handle(const Messages::WindowManagerServer::SetWindowTaskbarRect&) override;
+
```

Comment:
Please avoid making whitespace-only changes where there's nothing else being changed in the same file.